examples/async-graphql/Cargo.toml

```
@@ -8,6 +8,6 @@ publish = false
 axum = { path = "../../axum" }
 tokio = { version = "1.0", features = ["full"] }
 tracing = "0.1"
-tracing-subscriber = "0.2"
+tracing-subscriber = {version="0.3", features=["env-filter"]}
```

nitpick: We normally format the dependencies like this

```suggestion
tracing-subscriber = { version="0.3", features=["env-filter"] }
```

---

Ok, done